cloudflare_worker/worker/src/wasmFunctions.ts

```diff
@@ -14,7 +14,19 @@
  * limitations under the License.
  */
 
+// This variable is added by the runtime of Cloudflare worker. It contains the
+// binary data of the wasm file.
 declare var wasm: any;
+
+// `Wrangler` uses `wasm-pack build --target no-modules` [^1] to build wasm.
```

Comment:
Maybe lowercase the "w" since Unix commands are case-sensitive.

Reply:
Done